libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -114,9 +115,11 @@ export class Function extends cloud.FunctionBase {
       role: this.role.name,
     });
 
+    const functionName = this.sanitizeName(`${this.node.id}-${this.node.addr}`);
```

Comment:
I'd like to set the name via property.

Reply:
Same as `Counter`

`new cloud.Function() as "F1"`;

On AWS Console you will see `F1-<addr>`